docs/reference/announcements/880.md

```diff
@@ -115,6 +115,15 @@ Camunda 8.8 introduces a streamlined architecture, consolidating core components
 
 You can download the alpha release of the unified package from the Camunda GitHub repository, either as an executable Java application (Camunda Orchestration Core) or a Docker image.
 
+### Identity and authentication limitations in 8.8.0-alpha2
+
+The 13.0.0-alpha2 Helm chart released with Camunda 8.8.0-alpha2 establishes a new default setup to support 8.8 [Identity management updates](#identity-management-updates-saasself-managed). Currently, this setup is limited to the following components:
+
+- The Orchestration core (Zeebe, Operate, Tasklist, and Orchestration Identity)
```

Comment:
Does the "orchestration core" also include Optimize? That's what we have in glossary.md, but wasn't sure with how unique Optimize is.

Reply:
The slack conversation is here: https://camunda.slack.com/archives/C01H4NG9XDY/p1740070705216559

But I would hold off on any immediate follow-up based on this comment https://github.com/camunda/camunda-docs/pull/5133#discussion_r1983191790 - it sounds like Optimize will help take on that discussion.